code/src/components/DucklingHeader.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Lottie from 'lottie-react'
+import Duck from '../lotties/sassy-duck';
+
+export const DucklingHeader = () => {
+  const defaultOptions = {
+    loop: true,
+    autoplay: true,
+    animationData: Duck,
+    rendererSettings: {
+      preserveAspectRatio: 'xMidYMid slice'
```

Comment:
Nice first user experience of the game! Feels so cute and you really want to start the game to help the poor duckling.